### PR TITLE
Gather user data

### PR DESCRIPTION
**Pull request by metareven**
implemented a simple activity with an animation that is fired while gathering data about the user

https://jira.skunk-works.no/browse/IDBIO-36

How to test:
Fire up the app on your favorite real device (emulators do not support bluetooth)
Allow the application to use your camera etc (this should already be done in another activity when enrolling perhaps?)
Restart the application and see that you are now presented with a beautiful rotating star and see that the app does not crash. You should see information about the data gathering in your console if you are debugging

### CODE REVIEW

File: app/src/main/res/layout/activity_gather.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <ImageView
+        android:id="@+id/gatherAnimation"
+        android:layout_width="122dp"
+        android:layout_height="200dp"
+        android:visibility="visible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.498"
```
Comment (jorunfa) [inline app/src/main/res/layout/activity_gather.xml]: That is .002 of vertical center? If so, was it on purpose?

File: app/src/main/res/layout/activity_gather.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        app:srcCompat="@android:drawable/btn_star_big_on" />
+</android.support.constraint.ConstraintLayout>
```
Comment (jorunfa) [inline app/src/main/res/layout/activity_gather.xml]: No biggie, but do we want new line at end of file?

File: build.gradle [hunk 3]
```diff
@@ -10,6 +10,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.4'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.novoda:bintray-release:0.8.0'
```
Comment (jorunfa) [inline build.gradle]: Are we realising any binaries - do we need this (at the moment)?

File: app/build.gradle [hunk 4]
```diff
@@ -29,6 +38,7 @@ dependencies {
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'com.telenor.connect:connect-android-sdk:1.7.1'
+    implementation project(':possumgather')
```
Comment (jorunfa) [inline app/build.gradle:41]: Why don't we need `implementation project(':core')` here?

**Review by jorunfa — COMMENTED** [review]
I guess we agreed that we can add all the awesome possum stuff source for the prototype.
What parts of the pull request can be ignored? `core/` and `possumgather/`?

**Comment by ulfcarlin** [comment]
How to Write a Git Commit Message: https://chris.beams.io/posts/git-commit/

File: app/build.gradle (hunk 4 above)
Comment (metareven) [reply app/build.gradle:41]: I assume it is because possumgather has this in its build.gradle.

File: build.gradle (hunk 3 above)
Comment (metareven) [reply build.gradle]: Not really, but it was needed in order to keep the source code of possumgather and possumcore intact. Removed the dependency and just removed it from possumgather as well

File: app/src/main/res/layout/activity_gather.xml (hunk 2 above)
Comment (metareven) [reply app/src/main/res/layout/activity_gather.xml]: Not quite sure what you mean here, do you want me to end the section at the previous line?
There is not an empty line at the bottom of the file in my editor at least

File: app/src/main/res/layout/activity_gather.xml (hunk 1 above)
Comment (metareven) [reply app/src/main/res/layout/activity_gather.xml]: Nope. Fixed

File: app/src/main/res/layout/activity_gather.xml (hunk 2 above)
Comment (jorunfa) [reply app/src/main/res/layout/activity_gather.xml]: The question is if we should have a new line at the end of the file... it's convention in a lot of places.
Google gave me this: https://stackoverflow.com/a/729795/2148380

File: app/build.gradle (hunk 4 above)
Comment (jorunfa) [reply app/build.gradle:41]: Ok-dokey.

**Review by jorunfa — APPROVED** [review]
LGTM

**Comment by jorunfa** [comment]
See the newline comment, though.

**Pull request merged**